de.metas.swat/de.metas.swat.base/src/main/java/de/metas/request/api/IRequestTypeDAO.java

```diff
@@ -0,0 +1,37 @@
+package de.metas.request.api;
+
+import java.util.Properties;
+
+import org.adempiere.util.ISingletonService;
+import org.compiere.model.I_R_RequestType;
+
+/*
+ * #%L
+ * de.metas.swat.base
+ * %%
+ * Copyright (C) 2016 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+public interface IRequestTypeDAO extends ISingletonService
+{
+
```

Comment:
plese document which request type is the vendor one and which request type is the customer one. also, can it return null, or is there an exception in case the type is not unique?